src/Widgets/AddressWidget/AddressWidgetClass.js

```diff
@@ -6,6 +6,18 @@ const AddressWidget = Scrivito.provideWidgetClass('AddressWidget', {
     listItems: ['widgetlist', { only: 'AddressListItemWidget' }],
```

Comment:
`address` and `listItems` is no longer needed. Please remove them.

Reply:
Please also adjust the `registerTextExtract` to include the newly created attributes.